website/main.py

```diff
@@ -129,16 +155,39 @@ def update(attrname, old, new):
         target = data['sname'][selected_index]  # pulling out object name
 
         band, mags, magserr = genphot.main(target)  # grab photometric data fro given object
-        photcds.data['bands'] = band  # update photometric table with known bands
-        photcds.data['vals'] = mags  # update photometric table with corresponding magnitudes
-        photcds.data['valserr'] = magserr  # update photometric table with corresponding magnitude errors
+        photcds.data = dict(bands=band, vals=mags, valserr=magserr)
```

Comment:
couldn't remember which sources actually have photometry to check this is fine but it should be (and also stop a warning message on terminal); use identical method later which works

Reply:
It's fine, bar the issue with duplicates in #64